docs/notes.md

```diff
@@ -0,0 +1,23 @@
+# Notes on a "NextGen Universal Package"
+
+There's been some discussion in the community about the combinatorial effects of opening up the possibility to build ever-more-complicated systems while making it more difficult to resproduce results in the community, or to share results for integration into our collective understanding of the world. 
+
+The first step to building that collective understanding is based on having the same interpretation and categorization of the data we have and will produce. 
```

Comment:
The first step in building a shared understanding is to align on the interpretation and categorization of both existing and newly generated data.